spec/grid_spec.rb

```diff
@@ -2,75 +2,86 @@
 require_relative "../lib/grid.rb"
 
 RSpec.describe Grid do
-  describe "#coordinate_valid?" do
-    it "is true if in the first row and first column of the grid" do
+  describe "#coordinate_invalid?" do
```

Comment:
I think I prefer the `coordinate_valid?` as a method/concept. Making this about whether the data is invalid means it is facing and pandering to another object.